packages/Sortable/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@paprika/sortable",
+  "version": "0.1.0",
+  "description": "Sortable component",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/sortable"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/button": "^0.1.0",
+    "@paprika/icon": "^0.1.0",
+    "@paprika/l10n": "^0.1.0",
+    "@paprika/stylers": "^0.1.0",
+    "@paprika/tokens": "^0.1.0",
+    "prop-types": "^15.7.2",
+    "react-beautiful-dnd": "^11.0.4",
+    "uuid": "^3.3.2"
+  },
+  "peerDependencies": {
+    "react": "^16.4.0",
```

```suggestion
    "react": "^16.8.0",
```
mind upping this for hooks?